src/payment/checkout/payment-form/StripePaymentForm.jsx

```diff
@@ -81,12 +82,37 @@ function StripePaymentForm({
 
     setIsLoading(true);
 
-    const { error } = await stripe.confirmPayment({
+    const stripePaymentMethodHandler = async (result) => {
+      if (result.error) {
+        // Show error in payment form
+        if (result.error.type === 'card_error' || result.error.type === 'validation_error') {
+          setMessage(result.error.message);
+        } else {
+          setMessage('An unexpected error occurred.');
+        }
+        setIsLoading(false);
+      } else {
+        // Otherwise send paymentIntent.id to your server
+        // TODO: refactor to fetch in service.js
+        const postData = JSON.stringify({ payment_intent_id: result.paymentIntent.id });
+        await getAuthenticatedHttpClient()
```

Comment:
I've added this and the `.post()` instead of the below commented fetch because I was getting a 403 Forbidden. But I am uncertain about this